index.html

```diff
@@ -1521,6 +1479,55 @@ <h3>Hashlinks</h3>
 
     </section>
 
+    <section>
+      <h2>Zero-Knowledge Proofs</h2>
+      <em>This section is non-normative</em>
+        <p>
+The Verifiable Credentials Data Model is designed to be compatible with a
+variety of existing and emerging digital proof formats. Each
+proof format has benefits and drawbacks. Many proof formats cannot reveal
+selected attribute values in a verifiable credential; they can only reveal all
+(or none).
+        </p>
+        <p>
+Zero-Knowledge Proofs (ZKPs) are a proof format that enables
+privacy-preserving and data-minimization features in
+verifiable presentations, such as selective disclosure and predicate proofs.
+        </p>
+        <p>
+Selective disclosure is the ability of a holder to reveal a subset of
+the attributes of a verifiable credential in a verifiable presentation.
+A presentation based on zero-knowledge proof mechanisms only contains
+those attributes and associated values that are required to satisfy
+the presentation requirements.
+        </p>
+        <p>
+Zero-knowledge predicate proofs are a type of proof associated with
+an attribute. Predicate proofs include comparisons, such as
+"greater than", "less than", "not equal", "range",
+"set membership", and "set non-membership".
+Predicate proofs can be constructed without requiring the issuer
+to create special fields, such as
+"age-under-18", "age-over-21", or "age-over-60", in the verifiable credential
+at the time the credential was issued.
+For example, if a holder has a credential with a "birth-date" claim,
+the holder can create a predicate proof based on "birth-date" in the
+verifiable presentation. A predicate value derived from "birth-date" in
+the verifiable credential can cryptographically prove to the verifier that
+the holder's age is greater (or less) than a specified number, without
+revealing the holder's "birth-date".
+        </p>
+        <p>
+Verifiable credentials based on zero-knowledge proof mechanisms are
+also quantum-resistant after presentation.
+        </p>
+        <p>
+A drawback of zero-knowledge proof mechanisms is that they are
```

Comment:
`A drawback of zero-knowledge proof mechanisms is that they are`
-> `Drawbacks of zero-knowledge proof mechanisms include that they are`

Reply:
fixed in [5ff637c](https://github.com/w3c/vc-imp-guide/pull/32/commits/5ff637c784e5bec18463221499a55af27f8746e1)